source/installing-with-npm/index.html.md.erb

```diff
@@ -112,7 +112,7 @@ In your live application, we recommend [using an automated task or your build pi
 
 In your live application, we recommend:
 
-- [using an automated task or your build pipeline](../import-javascript/) instead of copying the files manually
+- [using an automated task or your build pipeline](../import-font-and-images-assets/) instead of copying the files manually
```

Comment:
The build pipeline link text is used in various places, but this one has the wrong link

Reply:
I think the original link was correct – this section is about 'Getting the JavaScript working'.

In step 2 we tell users to copy the minified JavaScript into their application. This is then saying that actually we'd rather they used an automated task to include the JS from the package, rather than copying it like this.